src/icmp/Icmp.cc

```diff
@@ -9,6 +9,8 @@
 /* DEBUG: section 37    ICMP Routines */
 
 #include "squid.h"
+#include "compat/socket.h"
```

Comment:
This looks like it can go now that `xclose()` is moved to `compat/unistd.h`
```suggestion
```

Reply:
removed